packages/stacked_tools/lib/src/templates/compiled_templates.dart

```diff
@@ -988,16 +1062,11 @@ void main() {
 
 // --------------------------------------------------
 
-// -------- Generic View Folder Template Path ----------
-
-const String kViewTemplateGenericFolderPath = 'lib/ui/views/generic/';
```

Comment:
@ferrarafer I added this for delete command i needed the folder path to remove the view and viewmodel